recipes/pyomo.solvers/meta.yaml

```diff
@@ -0,0 +1,36 @@
+# Do metapackages have a version?
+# Do metapackages have a license?
+
+package:
+  name: pyomo.solvers
+  version: 1.0
```

Comment:
Please use a jinja variable for the version.
(You check our example recipe on how to do that.)